src/ProvidesJsonAPI.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration;
+
+use JsonSerializable;
+
+interface ProvidesJsonAPI extends JsonSerializable
```

Comment:
Reviewer note: adding this JsonSerializable is a hack to cheat Symfony's http foundation classes to let a class be returned from a controller. By default, controllers are expected to return response objects, strings, arrays, JsonSerializables or null. If we return a single `Order` instance, then http foundation will not accept it unless it can be converted to a json string.

The `TransformsOneToJsonApi` middleware does not rely on the json serializable because it can actually extract the original response from the controller and then process it. So JsonSerializable here is only for passing through Symfony's http foundation gates.